usaspending_api/common/mixins.py

```diff
@@ -185,8 +185,8 @@ def build_response(self, request, *args, **kwargs):
         queryset = kwargs.get('queryset')
 
         # workaround to handle both GET and POST requests
-        params = dict(self.request.query_params)
-        params.update(dict(self.request.data))
+        params = self.request.query_params.copy()  # copy() creates mutable copy of a QueryDict
```

Comment:
Just noting this creates a shallow copy of the dict contents which I don't think _will_ matter but _may_